Common/src/main/java/com/tabnineCommon/binary/requests/config/StateResponse.kt

```diff
@@ -12,7 +12,11 @@ data class StateResponse(
     @SerializedName("installation_time")
     val installationTime: String? = null,
     @SerializedName("cloud_connection_health_status")
-    val cloudConnectionHealthStatus: CloudConnectionHealthStatus = CloudConnectionHealthStatus.Ok
+    val cloudConnectionHealthStatus: CloudConnectionHealthStatus = CloudConnectionHealthStatus.Ok,
+    @SerializedName("is_logged_in")
+    val isLoggedIn: Boolean? = null,
+    @SerializedName("user_name")
+    var userName: String? = null,
```

Comment:
isn't the `isLoggedIn` boolean redundant? isn't it equivalent to `userName != null`?

Reply:
we get it anyway from the binary

Reply:
so maybe worth removing from the binary as well🤷🏻‍♂️